core/domain/prod_validators.py

```diff
@@ -689,7 +449,7 @@ def _validate_node_count(
                     'collection_ids %s' % (
                         base_model_validators.ERROR_CATEGORY_FIELD_CHECK),
                     'Entity id %s: based on field collection_ids having'
-                    ' value %s, expected model %s with id %s but it doesn\'t'
+                    ' value %s, expect model %s with id %s but it doesn\'t'
```

Comment:
Why'd we make these changes? Maybe we can undo them? (including all of the similar changes below)